core/src/stored/backends/dedupable_device.d/bareos-sd.d/device/Dedupable.conf.example.in

```diff
@@ -9,14 +9,14 @@ Device {
 
   Device Options = "block size=4k"
 
-  ArchiveDevice = "path/to/storage"
+  Archive Device = @archivedir@
```

Comment:
Is it safe to use the same Archive Device path for normal file storage and dedupable file storage? If not, I would advise to set another path here.

Reply:
As already spoken it is possible to have both in the same place.